src/Detectors/Languages.php

```diff
@@ -26,4 +26,13 @@ public function detect(): array
 
         return $languages;
     }
+
+    private function isEmpty($value): boolval
```

Comment:
This function should be here https://github.com/oscarotero/Embed/blob/master/src/functions.php

The return type `boolval` doesn't exist. It should be `bool`.